NewHaven/test/TurnSequence.cpp

```diff
@@ -0,0 +1,105 @@
+//
+// Created by Muher on 2020-03-11.
+//
+
+#include <iostream>
+#include "Player.h"
+#include "Resources.h"
+
+using namespace std;
+
+int main(){
+    // *** This is the sequence of the currently active player ***
+
+    // Initiate decks
+    auto *bDeck = new BuildingDeck();
+    auto *hDeck = new HarvestDeck();
+
+    auto *player = new Player(); // Initiate player
+
+    // Start of the game, each player should have 6 buildings and 2 Harvest Tiles
+    for (int i = 0; i < 5; i ++)
+        (*player).drawBuilding(*bDeck->draw()); // Player draws a building from the deck
```

Comment:
You say each player should have 6 buildings, but proceed to only draw 5 buildings.